CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ version:
 
 - Users
   + [`ember new` diff](https://github.com/kellyselden/ember-cli-output/compare/v2.4.1...v2.4.2)
-  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + Upgrade your project's ember-cli version - [docs](http://ember-cli.com/#project-update)
```

Comment:
This should be `http://ember-cli.com/user-guide/#upgrading` (we changed the URL's a while back, but apparently didn't update these links).